prov/efa/src/rdm/efa_rdm_cq.c

```diff
@@ -476,6 +476,12 @@ static void efa_rdm_cq_handle_recv_completion(struct efa_ibv_cq *ibv_cq, struct
 		pkt_entry->peer->is_local = 0;
 	}
 
+
+	base_hdr = efa_rdm_pke_get_base_hdr(pkt_entry);
+	pkt_type = base_hdr->type;
+	if (pkt_type == EFA_RDM_EAGER_RTW_PKT)
```

Comment:
It is possible that pkt may not have hdr, use `efa_rdm_pkt_type_of_pke`

Reply:
Also we can make the whole if in `#if LTTNG` to save instructions for release build